docs/docs/reference/olap-engines/duckdb.md

```diff
@@ -25,4 +25,6 @@ Well now you can! For more details, see our page on [Using Multiple OLAP Engines
 
 ## Additional Notes
 
-- For dashboards powered by DuckDB, [measure definitions](/build/dashboards/dashboards.md#measures) are required to follow standard [DuckDB SQL](https://duckdb.org/docs/sql/introduction) syntax.
\ No newline at end of file
+- For dashboards powered by DuckDB, [measure definitions](/build/dashboards/dashboards.md#measures) are required to follow standard [DuckDB SQL](https://duckdb.org/docs/sql/introduction) syntax.
+- Known issue around creating a DuckDB via the UI, you will need to create the YAML file manually.
```

Comment:
Probably would clarify this is a DuckDB source

Reply:
This is for connecting to DuckDB as an OLAP engine... 
It's a bit confusing in general in the UI, is it a source or an engine?
Discussion on where this should be is already being discussed :)